### ISSUE TITLE
Error compiling build.cake

### ISSUE THREAD

**Issue by EdePC** [bfritscher/carnac#17]
```
PS C:\Users\EdSon\Desktop\carnac-3.1-beta> .\build.ps1
Preparing to run build script...
Running build script...
Analyzing build script...
Processing build script...
Installing tools...
Installing addins...
Compiling build script...
Error: C:/Users/EdSon/Desktop/carnac-3.1-beta/build.cake(15,21): error CS1525: Invalid expression term ''
```
Line 15: var githubRepoUrl = $"https://github.com/{githubRepo}";

I use Windows 10 lastest version

**Comment by bfritscher** [comment]
Yes, I did not use the original build scripts, and just compiled from Visual Studio. I have seen that the original repo changed some things. Does the the current original build for you? If so I can try to merge the update.

**Comment by Gordin** [comment]
I had the same problem. I merged `master` from the upstream repo into your `beta` branch and that made it work.
It's currently stuck at the unit tests and everything seems to be failing though...
The build is successful though and the resulting exe file seems to work

**Comment by EdePC** [comment]
I have exactly the same original request: https://github.com/bfritscher/carnac/issues/18
